docs/reference/runtimes/python/python-reference.md

```diff
@@ -54,20 +58,30 @@ ADD ./my-function.yaml /etc/nuclio/config/processor/processor.yaml
 CMD [ "processor" ]
 ```
 
-### Notes
+<a id="function-configuration"></a>
+## Function configuration
+
+Your function-configuration file (for example, `my-function.yaml` for the [example Dockerfile](#dockerfile)) must include the name of your handler function and Python runtime. For more information, see the [function-configuration reference](/docs/reference/function-configuration/function-configuration-reference.md). For example:
 
-- Make sure to change both `my-function-code` & `my-function.yaml` to your real function code & configuration
-- `my-function.yaml` should at least include your handler & runtime, in example:
 ```yaml
 spec:
   handler: main:handler
   runtime: python:3.6
+```
+
+<a id="build-and-execution"></a>
+## Build and execution
+
+Following are example commands for building and running the latest version of a `my-function` function that's listening on port 8090;
+replace the function name, version and the port number, as needed:
```

Comment:
1. Please change "replace the function name, version and the port number" to my original version &mdash; "replace the function name and version and the port number"; alternatively, you can use "replace the function name, function version, and port number", but the current version is grammatically incorrect.
2. Internal cosmetics: join lines 75 and 76 (i.e., "replace" should be just after "8090;", on the same line).